benchmark_runner/jupyterlab/templates/logs_operations/logs_operations.py

```diff
@@ -0,0 +1,92 @@
+
+import os
+
+# Progress bar
+import requests
+import ipywidgets as widgets
+from tqdm.auto import tqdm
+
+# Display progress bar
+from IPython.display import display
+
+# logging
+import logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+from typeguard import typechecked
+
+
+class LogsOperations:
+    """
+    This class responsible for logs operations
+    """
+    CHUNK_SIZE = 8192
+
+    def __init__(self, s3_logs_url: str):
+        # s3 Logs url is mandatory
+        if not s3_logs_url:
+            raise ValueError("s3_logs_url is empty")
+        self.s3_logs_url = s3_logs_url
+        self.logs_dir = os.path.join(os.path.join(os.getcwd(), 'logs'))
+        self.filename = self.s3_logs_url.split('/')[-1]
+        self.log_dir_path = os.path.join(self.logs_dir, self.filename)
+
```

Comment:
Can you make this variables private, if not using some other methods outside of this class

Reply:
I cannot make it private because I am using it in other [Module](https://github.com/redhat-performance/benchmark-runner/pull/608/files#diff-7a87d1123b275cd58cc16fae1a9ef3236f3e79310722d868170213a2931d4fb7)

Reply:
ok